CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to GlueStick
+
+GlueStick is one of TrueCar's open source projects that is build to speed up the development process internally in [truecar.com](https://truecar.com) and also help a larger audience reduce the time needed to build apps. It allows the developer to create and deploy enterprise-grade React and Redux application right from your command line.
```

Comment:
Try `Gluestick is an open source project sponsored by TrueCar (link) that uses it to speed up the...